xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -349,3 +366,25 @@ void CNetworkAndroid::RetrieveInterfaces()
                 n.toString());
   }
 }
+
+void CNetworkAndroid::onAvailable(const CJNINetwork n)
+{
+  CLog::Log(LOGDEBUG, "CNetworkAndroid::onAvailable The default network is now: {}", n.toString());
+
+  CJNIConnectivityManager connman(CXBMCApp::getSystemService(CJNIContext::CONNECTIVITY_SERVICE));
```

Comment:
```suggestion
  CJNIConnectivityManager connMan{CXBMCApp::getSystemService(CJNIContext::CONNECTIVITY_SERVICE)};
```

Reply:
Both are valid ways to call the constructor, whatever you want